packages/client/src/raffles/types/rafflesQuery.types.ts

```diff
@@ -0,0 +1,13 @@
+import type { RaffleStatus } from './raffles.types';
+
+export type RafflesQuery = {
+  status: RaffleStatus;
```

Comment:
Is `status` required?

Reply:
no, I already update it to optional. pls double check